pkg/api/api.go

```diff
@@ -80,6 +80,8 @@ func (hs *HTTPServer) registerRoutes() {
 	r.Get("/live/pipeline", reqGrafanaAdmin, hs.Index)
 	r.Get("/live/cloud", reqGrafanaAdmin, hs.Index)
 
+	r.Get("/chat", reqSignedIn, hs.Index)
```

Comment:
lets remove this to start -- org wide chat seems too overwhelming!

Reply:
Removed